src/controller/python/chip-repl.py

```diff
@@ -62,6 +62,29 @@ def main():
 if device.needsNetworkCredentials:
   device.ConnectToWifi("ssid", "password")
 
+######## Thread provisioning ########
```

Comment:
Out of curiosity, how are these changes related to `chip-device-ctrl.py` and `ChipDeviceCtrl.py`? I'm a bit confused which controller solution we should put effort in as after not-fully-completed switch from the native chip-tool to the Python controller it seems that new API within the Python controller is emerging. Is the new API going to replace bindings from `ChipDeviceCtrl.py` at some point?

Reply:
I am trying to get a scriptable environment - I have found tha chip-device-ctrl relies on interactive use and I would like to aim for either interactive (ipython) or automated scripting use. 

My long term goal would be to replace/remove device-ctrl completely and only keep the repl part.